include/service/entities/content/article_service_v2.js

```diff
@@ -34,18 +34,19 @@ module.exports = function(pb) {
      * @constructor
      * @extends BaseObjectService
      */
-    function ArticleServiceV2(context){
+    function ArticleServiceV2(context, options){
```

Comment:
There is no need for an options object.  Use the context parameter to provide site which is probably already in the context object.